### ISSUE TITLE
Support for new labels and new models

### ISSUE THREAD

**Issue by saitej123** [urchade/GLiNER#24]
any new model released for these entities: 
"PHONE NUMBER"
"EMAIL ADDRESS"
"LOCATION": 
"IP ADDRESS"
"PASSPORT NUMBER"
"CREDIT CARD NUMBER"

Base model currently not performing to be mark with this text:
"John Smith, residing at 123 Main Street, New York, NY, USA, can be reached at 555-123-4567 or via email at john.smith@example.com. His passport number is AB123456, and his credit card number is 1234 5678 9012 3456."

**Comment by urchade** [comment]
You can try "urchade/gliner_large" which is the most powerful currently.

![image_2024-03-17_144147196](https://github.com/urchade/GLiNER/assets/38214774/62e1c41a-4622-4b93-849f-bbe608f78746)

**Comment by saitej123** [comment]
benchmarks available for large vs large v2 on ontonotes ?

**Comment by saitej123** [comment]
emial not capturing sometimes , nost sure reason !
<img width="422" alt="image" src="https://github.com/urchade/GLiNER/assets/18657211/93b6e482-c01b-4688-b4f7-57ee25980edf">

**Comment by urchade** [comment]
Did you try lower casing labels ?

**Comment by urchade** [comment]
Large is bette than la large v2 in général

I will publish a benchmark

**Comment by saitej123** [comment]
some wrong captures  ,it works with lower case also
<img width="431" alt="image" src="https://github.com/urchade/GLiNER/assets/18657211/16fe8158-b55f-4673-a2e5-250a2a4a9fff">

**Comment by saitej123** [comment]
<img width="463" alt="image" src="https://github.com/urchade/GLiNER/assets/18657211/de44f51c-4fd3-473f-89b1-180a957d7c3c">

another weird tagging

**Comment by urchade** [comment]
ok, I think you can try fine-tuning a model, even with very few samples (<10). It should improve the result for your use case. Look: https://github.com/urchade/GLiNER/blob/main/examples/finetune.ipynb

As reported in the paper, zero-shot results are still far from perfect (https://arxiv.org/pdf/2311.08526.pdf)

**Comment by saitej123** [comment]
Thanks @urchade  !

**Comment by CharlesOkwuagwu** [comment]
> emial not capturing sometimes , nost sure reason ! <img alt="image" width="422" src="https://private-user-images.githubusercontent.com/18657211/313482639-93b6e482-c01b-4688-b4f7-57ee25980edf.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTE5MjIzNDEsIm5iZiI6MTcxMTkyMjA0MSwicGF0aCI6Ii8xODY1NzIxMS8zMTM0ODI2MzktOTNiNmU0ODItYzAxYi00Njg4LWI0ZjctNTdlZTI1OTgwZWRmLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDAzMzElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwMzMxVDIxNTQwMVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTYxZTg3ODgyNTMwN2Y1YTVkMzU1ODU1ZDNjY2I1Mjc1MTUzOGZhZTBkYzI1ODQ3OTBlYmZlODY4NTJjMmI1OWEmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.MKuJjac_8N79oA7LC2F4z9DOest1EEnf4OibZmOFjt8">

hi @saitej123, Please what tool do you use to get the highlighted tags and data, thanks.

**Comment by saitej123** [comment]
Spacy with streamlit
